src/App.tsx

```diff
@@ -10,33 +12,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab): void => {
```

Comment:
```suggestion
  const handleTabSelected = (tab: Tab): void => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364